cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserPermissionsManager.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2023 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.servlet;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.stream.Collectors;
+import java.util.Collections;
+import com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig;
+import org.eclipse.jetty.security.UserStore;
+import org.eclipse.jetty.security.PropertyUserStore;
+import org.eclipse.jetty.security.AbstractLoginService;
+import com.linkedin.kafka.cruisecontrol.config.constants.WebServerConfig;
+import javax.security.auth.Subject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The util class for User Permissions
+ */
+public class UserPermissionsManager {
+    private static final Logger LOG = LoggerFactory.getLogger(UserPermissionsManager.class);
+
+    private final KafkaCruiseControlConfig _config;
+    private final Map<String, Set<String>> _rolesPerUsers;
+
+    public UserPermissionsManager(KafkaCruiseControlConfig config) {
+        _config = config;
+        _rolesPerUsers = createRolesPerUsersMap();
+    }
+
+    /** Builds a map between existing users and their roles in Cruise Control
```

Comment:
nit: usually the comments start from the second line. Let's make it consistent.

```
/**
 * some comment
*/
```

Reply:
Ack